hnn_core/tests/conftest.py

```diff
@@ -63,3 +68,45 @@ def pytest_runtest_setup(item):
             # and test name
             if test_name is not None:
                 pytest.xfail("previous test failed ({})".format(test_name))
+
+
+@pytest.fixture(scope='module')
+def run_hnn_core():
```

Comment:
what about putting "fixture" in this function name? Otherwise it looks like a mysterious parameter to other testing functions.

Reply:
+1 @kohl-carmen can you make the changes yourself?